game-frontend/src/Game.tsx

```diff
@@ -26,27 +27,47 @@ function Game(): JSX.Element {
   const gameState = useGameState();
   const userContext = useUserContext();
   const role = useRole();
-  const playStartSound = useStartSound();
+
+  const matchIsRunning =
+    gameState.objectIds
+      .map(id => gameState.objectMap[id])
+      .filter(gameObject => gameObject.type === GameObjectTypes.Unit).length >
+    0;
 
   const isPlayer = role === Role.PLAYER;
   const isLiving =
-    isPlayer && userContext && !!gameState.objectMap[userContext.unitId];
+    isPlayer && matchIsRunning && !!gameState.objectMap[userContext.unitId];
+  const isOnlyOnePlayerLeft =
+    gameState.objectIds
+      .map(id => gameState.objectMap[id])
+      .filter(gameObject => gameObject.type === GameObjectTypes.Unit).length ===
+    1;
   const isDead = isPlayer && hasJoinedGame && !isLiving;
+  const hasWon = isPlayer && hasJoinedGame && isLiving && isOnlyOnePlayerLeft;
+
+  console.log({
```

Comment:
Not quite sure how ready for review this PR is? :)

Reply:
I wanted @sir-marc to have a look at the front-end before finishing and cleaning it up :)

Reply:
Btw, how can you set the PR as a draft?

Reply:
Upon creation you can choose whether to create a PR or create a draft PR - but afaik you cannot switch it as soon as it's created (or at least I don't know how 😅 )

Reply:
There's a small link below the assigned reviewers. Didn't catch that before ^^